src/components/date/date.style.js

```diff
@@ -23,6 +26,19 @@ const StyledDateInput = styled.div`
       margin-right: -8px;
     }
   }
+
+  ${({ applyDateRangeStyling, size, labelInline }) =>
+    applyDateRangeStyling &&
+    !labelInline &&
+    css`
+      ${FieldLineStyle} {
+        max-width: ${datePickerWidth[size]};
+      }
+
+      ${StyledValidationMessage}, ${StyledLabel} {
+        word-break: break-word;
```

Comment:
looking up `word-break` on MDN, it says `break-word` is deprecated - so I'm nervous about including it in new code even though it's still likely to work in most/all browsers. It looks like `overflow-wrap: anywhere` will have the same effect but that's from about a minute of investigation so don't take my word for it!

Reply:
Yeah good spot, I'll get that updated thanks 👍